src/earn/PoolCard.tsx

```diff
@@ -0,0 +1,193 @@
+import BigNumber from 'bignumber.js'
+import React, { useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import Button, { BtnSizes, BtnTypes, TextSizes } from 'src/components/Button'
+import TokenDisplay from 'src/components/TokenDisplay'
+import TokenIcon from 'src/components/TokenIcon'
+import { Pool } from 'src/earn/types'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { useSelector } from 'src/redux/hooks'
+import { NETWORK_NAMES } from 'src/shared/conts'
+import { Colors } from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { tokensByIdSelector } from 'src/tokens/selectors'
+import { TokenBalance } from 'src/tokens/slice'
+export default function PoolCard({ pool, testID = 'PoolCard' }: { pool: Pool; testID?: string }) {
+  const { tokens, networkId, poolTokenId, depositTokenId } = pool
+  const { t } = useTranslation()
+  const allTokens = useSelector((state) => tokensByIdSelector(state, [networkId]))
+  const tokensInfo = useMemo(() => {
+    return tokens
+      .map((tokenId) => allTokens[tokenId])
+      .filter((token): token is TokenBalance => !!token)
+  }, [pool.tokens, allTokens])
+  const poolTokenInfo = allTokens[poolTokenId]
+  const depositTokenInfo = allTokens[depositTokenId]
```

Comment:
can we use `useTokenInfo` instead of this custom code? troopo needed this because it supports an array of tokens vs a single deposit / withdraw token